tensilelite/rocisa/rocisa/src/code.cpp

```diff
@@ -166,7 +172,13 @@ void init_code(nb::module_ m)
         .def("count", &rocisa::Module::count)
         .def("getItem", &rocisa::Module::getItem)
         .def("setItem", &rocisa::Module::setItem)
-        .def("setItems", &rocisa::Module::setItems)
+        // nanobind cannot reliably bind the same method signatures that differ
```

Comment:
I ran clang-format in my PR and found this file is modified. Can you run clang-format?

Reply:
We also have a pre-commit file in the hooks folder in hipblaslt.

Reply:
done.